docs/guides/testing-single-file-components-with-jest.md

```diff
@@ -10,6 +10,12 @@ Jest is a test runner developed by Facebook, aiming to deliver a battery-include
 
 We will assume you are starting with a setup that already has webpack, vue-loader and Babel properly configured - e.g. the `webpack-simple` template scaffolded by `vue-cli`.
 
+> Alternatively, if you are using the Vue CLI rather than webpack to build your project, you can use the plugin [cli-plugin-unit-jest](https://cli.vuejs.org/core-plugins/unit-jest.html#injected-commands) to run Jest tests.
```

Comment:
Vue CLI does use webpack internally to build the project, though. I think this is a bit confusing. Maybe I misunderstood - can you potentially reword this?

Reply:
For sure, I was aware of that and it is partly where I got confused as I just followed the instructions on my first try.

I could take it out:

> Alternatively, if you are using the Vue CLI to build your project , you...

Or, making it more explicit:

> Alternatively, if you are using the Vue CLI to build your project (where you don't have a separate webpack config and your babel config is using the preset `@vue/cli-plugin-babel/preset`), you....

As I understand it, the guide is written from the perspective of having used a preset that defaults to using webpack/you have ejected from the Vue CLI and so you would doing something like `npm webpack...` rather than `npm vue-cli-service`? But if you are using the CLI Service you get some of the config essentially for free and you can skip those sections.

Reply:
I am definitely in favor of encouraging people to use the cli. I see two groups of users:

- beginner/intermediate, who just want to test their components. They should use the CLI.
- advanced users, who want to configure things for more control. They are experienced with the toolchain, they will figure it out.

Reply:
So following that, we could go for the most straight forward option:

> Alternatively, if you are using the Vue CLI to build your project , you...

People should know if they are using the CLI and so read on and advanced users can work out what applies to them. I'll push that in, but please let me know if you would like further changes.

Reply:
Thank you for your contribution!